python/hsfs/core/monitoring_window_config.py

```diff
@@ -67,7 +67,7 @@ def __str__(self):
         return self.json()
 
     def __repr__(self):
-        return json.dumps(humps.decamelize(self.to_dict()), indent=2)
+        return f"MonitoringWindowConfig(type: {self._window_config_type})"
```

Comment:
I don't think we should make it harder to introspect entities quickly, even for us when we need to do debugging

Reply:
For debugging purposes, I agree. But for introspection I think it's better to have a specific .describe() method.